### ISSUE TITLE
Slack notification failing when image url is used for icon

### ISSUE THREAD

**Issue by arakir** [Tautulli/Tautulli#666]
When I set "Slack Icon" to ie: "http://qnapclub.eu/custom_pic_qpkg/269.png" and hit Test notification, it shows "500 internal error" in lower right corner as a result.

When I tried to send the request manually using payload bellow, it works for me.
Payload I used:

```
{
  "text":"Test",
  "username":"PlexPy", 
  "icon_url":"http://qnapclub.eu/custom_pic_qpkg/269.png"
}
```

Branch: master
Commit: 786a374
Platform: Windows 7
Python Version: 2.7.6 (default, Nov 10 2013, 19:24:18) [MSC v.1500 32 bit (Intel)]

**Comment by JonnyWong16** [comment]
You found a bug. Thanks.

This line: https://github.com/drzoidberg33/plexpy/blob/master/plexpy/notifiers.py#L1779
should be:

```
data['icon_url'] = self.icon_emoji
```

**Comment by JonnyWong16** [comment]
Fixed in v1.3.15.
